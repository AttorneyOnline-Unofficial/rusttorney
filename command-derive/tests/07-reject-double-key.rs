use command_derive::*;

#[derive(Command)]
enum ClientRequest {
    #[command(code = "HI", handle = "handle_handshake")]
    #[command(code = "HI")]
    Handshake {
        hdid: String
    }
}

fn main() {}
