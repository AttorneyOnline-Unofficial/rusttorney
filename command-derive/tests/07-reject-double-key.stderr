error: Keys in `#[command(...)]` can't be repeated
 --> $DIR/07-reject-double-key.rs:3:10
  |
3 | #[derive(Command)]
  |          ^^^^^^^
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
